release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -274,6 +275,27 @@ For more information, see xref:../networking/metallb/metallb-frr-k8s.adoc#metall
 === AdminNetworkPolicy is generally available
 This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Before namespaces are created, cluster Administrators can use ANP and BANP to apply cluster-scoped network policies and safeguards for an entire cluster. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale without having to duplicate their network policies on each namespace.
 
+For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
+
+[id="ocp-4-16-networking-sdn-ovnk-live-migration_{context}"]
+==== Live migration to the OVN-Kubernetes network plugin
+
+Previously, when migrating from OpenShift SDN to OVN-Kubernetes, the only available option was an _offline_ migration method. This process included some downtime, during which clusters were unreachable. 
+
+With this update, a _live_ migration method has been introduced. The live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and Azure Red Hat OpenShift deployment types. It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
```

Comment:
Rephrasing suggestion:

This release introduces a _live_ migration method, where the OpenShift SDN network plugin and its network configurations, connections, and associated resources are migrated to the OVN-Kubernetes network plugin without service interruption.

- Otherwise I'd just remove the comma after "connections" :)